[31;1mCommandError: Unable to serialize database: cursor "_django_curs_139760752969536_sync_3" does not exist
[0m[{"model": "contenttypes.contenttype", "pk": 1, "fields": {"app_label": "admin", "model": "logentry"}}, {"model": "contenttypes.contenttype", "pk": 2, "fields": {"app_label": "auth", "model": "permission"}}, {"model": "contenttypes.contenttype", "pk": 3, "fields": {"app_label": "auth", "model": "group"}}, {"model": "contenttypes.contenttype", "pk": 4, "fields": {"app_label": "contenttypes", "model": "contenttype"}}, {"model": "contenttypes.contenttype", "pk": 5, "fields": {"app_label": "sessions", "model": "session"}}, {"model": "contenttypes.contenttype", "pk": 6, "fields": {"app_label": "api", "model": "category"}}, {"model": "contenttypes.contenttype", "pk": 7, "fields": {"app_label": "api", "model": "genre"}}, {"model": "contenttypes.contenttype", "pk": 8, "fields": {"app_label": "api", "model": "title"}}, {"model": "contenttypes.contenttype", "pk": 9, "fields": {"app_label": "api", "model": "review"}}, {"model": "contenttypes.contenttype", "pk": 10, "fields": {"app_label": "api", "model": "comment"}}, {"model": "contenttypes.contenttype", "pk": 11, "fields": {"app_label": "auth_user", "model": "customuser"}}, {"model": "sessions.session", "pk": "aigwlli37v3x5ftlgc04vd8mqn2qwr4y", "fields": {"session_data": "MzNlYmU4YTVmOGIzOTQxZmZmMDIyODVhZmI1NjYzMGM2OWY3OWQxMjp7Il9hdXRoX3VzZXJfaWQiOiIxIiwiX2F1dGhfdXNlcl9iYWNrZW5kIjoiZGphbmdvLmNvbnRyaWIuYXV0aC5iYWNrZW5kcy5Nb2RlbEJhY2tlbmQiLCJfYXV0aF91c2VyX2hhc2giOiJkNDJjYTBjYjk0MTg3NTFlYTgyMWEwYzk4MTZiNzA4YmYyMTQ1ODQ4In0=", "expire_date": "2021-05-11T23:33:59.404Z"}}